### ISSUE TITLE
Process finished with exit code -1073741819 (0xC0000005)

### ISSUE THREAD

**Issue by RomanticLeon** [WenmuZhou/PSENet.pytorch#82]
在运行train.py进行训练时，当使用gpu时，运行到loss.backward时就会报出 Process finished with exit code -1073741819 (0xC0000005)。我是在pycharm中运行的代码，参数配置如下：
2020-06-09 17:24:46 INFO      utils.py: logger init finished
2020-06-09 17:24:46 INFO      train.py: {'Lambda': 0.7,
 'OHEM_ratio': 3,
 'backbone': 'resnet50',
 'checkpoint': '',
 'data_shape': 32,
 'display_input_images': False,
 'display_interval': 10,
 'display_output_images': False,
 'end_lr': 1e-07,
 'epochs': 600,
 'gpu_id': '0',
 'lr': 0.0001,
 'lr_decay_step': [200, 400],
 'lr_gamma': 0.1,
 'm': 0.5,
 'n': 6,
 'output_dir': './output/output_train_log',
 'pretrained': True,
 'restart_training': True,
 'scale': 1,
 'seed': 2,
 'show_images_interval': 50,
 'start_epoch': 0,
 'testroot': '../../OCR_dataset/ICDAR2015/test',
 'train_batch_size': 2,
 'trainroot': '../../OCR_dataset/ICDAR2015/train',
 'warm_up_epoch': 6,
 'warm_up_lr': 1e-05,
 'weight_decay': 0.0005,
 'workers': 8}
2020-06-09 17:24:46 INFO      train.py: train with gpu 0 and pytorch 1.5.0+cu92
2020-06-09 17:24:47.148053: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2020-06-09 17:24:49 INFO      resnet.py: load pretrained models from imagenet
2020-06-09 17:24:50 INFO      train.py: train dataset has 1000 samples,500 in dataloader

然而我在cpu上就可以正常运行，gpu上就不行，有人知道怎么解决吗？
